src/app/components/character/character.component.spec.ts

```diff
@@ -2,6 +2,7 @@ import { HtmlParser } from '@angular/compiler';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { CharacterComponent } from './character.component';
+import characterArray from '../../Character.json';
```

Comment:
This JSON file shouldn't be capitalized. It should also live inside some test folder in a subfolder labeled `data/` or `fixtures`